utils/check_copies.py

```diff
@@ -70,6 +70,14 @@
             " {paper_authors}.{supplements}"
         ),
     },
+    "README_es.md": {
+        "start_prompt": "🤗 Transformers actualmente proporciona las siguientes arquitecturass",
```

Comment:
It looks like the util that checks for copies does not find this start prompt in the README in Spanish. Could you make sure it is correct?